src/Controls/src/Core/Entry/Entry.iOS.cs

```diff
@@ -16,6 +16,11 @@ public static void MapAdjustsFontSizeToFitWidth(IEntryHandler handler, Entry ent
 		public static void MapText(IEntryHandler handler, Entry entry)
 		{
 			Platform.TextExtensions.UpdateText(handler.PlatformView, entry);
+
+			// Any text update requires that we update any attributed string formatting
+			handler.UpdateValue(nameof(IEntry.MaxLength));
+			handler.UpdateValue(nameof(IEntry.CharacterSpacing));
+			handler.UpdateValue(nameof(IEntry.HorizontalTextAlignment));
```

Comment:
I'm really new to the MAUI code (I really appreciate all your great work!), but I'm curious why we wouldn't have an `handler.UpdateValue(nameof(IEntry.TextColor));` here? I'm curious how these length, spacing and alignment attributes fix the text color (from what I gather, it looks like perhaps we are almost "tricking" iOS into updating the field?). Thank you! 😃